src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__length--31">
+      <div class="calendar__day calendar__day--sun"></div>
```

Comment:
start-day should be a modifier for the calendar block
```suggestion
    <div class="calendar calendar__length--31  calendar--start-day_sun">
      <div class="calendar__day"></div>
```